agilent_infiniium_scope.py

```diff
@@ -72,319 +72,350 @@ def channel_infoNONEXISTANT(self, c, channel):
         Tuple of (probeAtten, termination, scale, position, coupling, bwLimit, invert, units)
         """
         raise Exception('Not yet implemented')
-        #NOTES
-        #The scope's response to 'CH<x>?' is a string of format
-        #'1.0E1;1.0E1;2.5E1;0.0E0;DC;OFF;OFF;"V"'
-        #These strings represent respectively,
-        #probeAttenuation;termination;vertScale;vertPosition;coupling;bandwidthLimit;invert;vertUnit
+        """
+        NOTES
+        The scope's response to 'CH<x>?' is a string of format
+        '1.0E1;1.0E1;2.5E1;0.0E0;DC;OFF;OFF;"V"'
+        These strings represent respectively,
+        probeAttenuation;termination;vertScale;vertPosition;coupling;
+        bandwidthLimit;invert;vertUnit
```

Comment:
Makes sense.   I checked the manual and these commands are not even really supported by these scopes--i.e. AC/DC coupling is only available through probes.

I have removed all the non-implemented functions.